src/api.ts

```diff
@@ -22,6 +22,23 @@ export class Api {
       });
   }
 
+  sendLineOfCode(userId: string, lineOfCode: number): void {
+    const timestamp: number = moment().valueOf();
+    const body: RequestOptionsBody = {
+      userId,
+      timestamp,
+      lineOfCode
+    };
+
+    this.post(Endpoint.SEND_LINE_OF_CODE, 'POST', body)
+      .then((response: {}) => {
+        console.log('response', response);
```

Comment:
Maybe the first log parameter can be more explicit given the request done than just 'response' like "Send line of code response"